common/models/acl.js

```diff
@@ -378,6 +379,18 @@ module.exports = function(ACL) {
   };
 
   /**
+   * Test if ACL permission is ALLOW
+   * @returns {Boolean} true if ACL permission is ALLOW
+   */
+  ACL.prototype.isAllowed = function(defaultPermission) {
+    var permission = this.permission;
+    if (permission === ACL.DEFAULT) {
+      permission = defaultPermission || ACL.ALLOW;
+    }
+    return permission !== loopback.ACL.DENY;
+  };
```

Comment:
I changed the implem of this new helper following our discussion so to honour the defaultPermission setting for the given model